docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/signed-content.md

```diff
@@ -0,0 +1,166 @@
+
+  ```bash
+  uname -m
+  ```
+
+- Minimum hardware configuration of the Linux machine:
+
+  - 4 CPU
+  - 8 GB memory
+  - 150 GB storage. If you plan on using a content bundle, the actual storage will depend on the size of the content
+    bundle you will use to build the Edge installer ISO image.
+
+## Embed a Public Key in Edge Artifacts
+
+1. Clone the **CanvOS** repository.
+
+   ```shell
+    git clone https://github.com/spectrocloud/CanvOS.git
+   ```
+
+2. Change to the **CanvOS/** directory.
+
+   ```shell
+   cd CanvOS
+   ```
+
+3. View the available git tag.
+
+   ```shell
+   git tag
+   ```
+
+4. Check out the latest available tag. This guide uses the tag v4.4.a as an example.
+
+   ```
+   git checkout v4.4.a
+   ```
+
+5. In **CanvOS**, create a file named **.edge_custom_config.yaml**.
+
+6. Populate the YAML file with the following content. Replace the value for `base64EncodedValue` with the base64 encoded
+   value of your public key. The PEM format is base64 encoded. If you have your public key in the PEM format, you only
+   need to copy the base64 portion of the key, without the header nor footer.
+
+   ```yaml
+   content:
+       signing:
+           publicKey:
+             base64EncodedValue: *******
+             description: "This is a public key used for verifying content bundles and cluster definitions." 
+   ```
+
+7. In your **.arg** file, add the following parameter `EDGE_CUSTOM_CONFIG` and provide the path to your
+   **.edge_custom_config.yaml** file.
+
+   ```text {12}
+    CUSTOM TAG=demo
+    IMAGE_REGISTRY=ttl.sh
+    OS_DISTRIBUTION=ubuntu
+    IMAGE REPO=$0S_DISTRIBUTION
+    OS_VERSION=22.04
+    K8S_DISTRIBUTION=k35
+    ISO_NAME=palette-edge-installer
+    ARCH=amd64
+    UPDATE KERNEL=false
+    CLUSTERCONFIG=spc.tgz
+    CIS HARDENING=false
+    EDGE_CUSTOM_CONFIG=.edge-custom-config.yaml
+   ```
+
+8. Finish the rest of the EdgeForge process to build either the installer ISO or provider images. For more information,
+   refer to [Build Installer ISO](./build-installer-iso.md) and [Build Provider Images](./build-provider-images.md).
+
+   :::info
+
+   When building the installer ISO, you must set the `installationMode` parameter to `airgap` in your Edge installer
+   configuration user data. This ensures that your Edge host does not try to register itself with Palette, as this
+   feature is not available to connected clusters. For more information, refer to
+   [Installer Reference](../../edge-configuration/installer-reference.md#install-mode).
+
+   :::
+
+## Rotate or Remove Key
+
+9. (Optional) To rotate or remove the public key used by your Edge host, build a new provider image with the new key or
+   with no keys, and then create a cluster with that provider image.
+
+   If a cluster is created with a provider image with a new key, the new key will replace the old key. If a cluster is
+   created with a provider image with no keys, then the existing key will be removed.
+
+## Validate
+
```

Comment:
Let's call out the tabs in a sentence. Mainly because they can be easy to miss (per feedback)